server/auth-router.js

```diff
@@ -101,9 +101,14 @@ module.exports = function authRouter(config = {}) {
 		info(`LoginUI: execute logout, session id:${req.sessionID}, cookie:${getSyncCookie(req)}, user id:${req.user && req.user.id}`); // eslint-disable-line max-len
 		const returnUrl = encodeURIComponent(`https://${config.host}`);
 		const logoutUrl = `https://${config.auth0.domain}/v2/logout?returnTo=${returnUrl}`;
-		req.logout(); // removes req.user
-		noteLoggedOut(res);
-		res.redirect(logoutUrl);
+		req.logout({}, err => {
+			if (err) {
+				error('LoginUI: logout callback error:', err);
+			}
+			// removes req.user
```

Comment:
This comment seems to imply that calling `noteLoggedOut(res)` will remove `req.user`, but it's actually the call to req.logout that does that.

Reply:
good point will move up to correlate to the proper call